content/docs/dvclive/ml-frameworks/xgboost.md

```diff
@@ -14,9 +14,16 @@ from dvclive.xgb import DVCLiveCallback
 
 ...
 
+params = {
+  "eval_metric": "rmse"
+}
+
 xgboost.train(
-    param, dtrain, num_round=5, evals=[(dval, "eval_data")]
+    params,
+    dtrain,
+    num_round=5,
     callbacks=[DVCLiveCallback("eval_data")],
+    evals=[(dval, "eval_data")],
```

Comment:
Why do we use the name `eval_data` on both of these lines? Since I don't think they represent the same thing, I find it a bit confusing. You didn't introduce this, but I wonder if we should just drop `eval_data` from the `DVCLiveCallback`?

Also, any reason you moved `evals` below `callbacks`?

Reply:
Dropped "eval_data" from DVCLiveCallback() and made `callbacks` to be the last parameter.